expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
+
+about:
+  home: http://expat.sourceforge.net
+  license: MIT
+  summary: |
```

Comment:
I use it to make a multiline string: http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

It seems that `>` is better here, though.